provisioning/device/package.json

```diff
@@ -30,7 +30,7 @@
     "alltest": "istanbul cover node_modules/mocha/bin/_mocha -- --reporter spec test/_*_test*.js",
     "ci": "npm -s run lint && npm -s run build && npm -s run alltest-min && npm -s run check-cover",
     "test": "npm -s run lint && npm -s run build && npm -s run unittest",
-    "check-cover": "istanbul check-coverage --statements 77 --branches 65 --lines 79 --functions 69"
+    "check-cover": "istanbul check-coverage --statements 76 --branches 65 --lines 77 --functions 69"
```

Comment:
>-functions 6 [](start = 86, length = 12)

Consider adding scripts that call out the individual tests as was done for the hub e2e tests.  Could make debugging a bit easier.